File: src/main/java/io/github/tanguygab/conditionalactions/commands/ReloadCommand.java
package io.github.tanguygab.conditionalactions.commands;

import io.github.tanguygab.conditionalactions.ConditionalActions;
import org.bukkit.command.CommandSender;

import java.util.List;

public class ReloadCommand extends CACommand {

    public ReloadCommand(ConditionalActions plugin) {
        super(plugin);
    }

    @Override
    public void onCommand(CommandSender sender, String[] args) {
        if (!sender.hasPermission("conditionalactions")) {
            sendMessage(sender,"&cYou don't have the permission to do this!");
            return;
        }
        plugin.onDisable();
        plugin.onEnable();
        sendMessage(sender,"&aPlugin reloaded successfully!");
    }

    @Override
    public List<String> onTabComplete(CommandSender sender, String[] args) {
        return List.of();
    }
}

File: src/main/kotlin/io/github/tanguygab/conditionalactions/commands/ReloadCommand.kt
package io.github.tanguygab.conditionalactions.commands

import io.github.tanguygab.conditionalactions.ConditionalActions
import org.bukkit.command.CommandSender

class ReloadCommand(plugin: ConditionalActions) : CACommand(plugin) {
    override fun onCommand(sender: CommandSender, args: List<String>) {
        if (!sender.hasPermission("conditionalactions")) {
            sendMessage(sender, "&cYou don't have the permission to do this!")
            return
        }
        plugin.onDisable()
        plugin.onEnable()
        sendMessage(sender, "&aPlugin reloaded successfully!")
    }

    override fun onTabComplete(sender: CommandSender, args: List<String>) = listOf<String>()
}
